model.go

```diff
@@ -1127,6 +1128,7 @@ func modelV2Fields() (schema.Fields, schema.Defaults) {
 	fields["sla"] = schema.FieldMap(
 		schema.Fields{
 			"level":       schema.String(),
+			"owner":       schema.String(),
```

Comment:
Doesn't this mean that you need to introduce a new version of the model struct? (v3 by the looks)